Recommender System via a simple Matrix Factorization

kamilc opened the pull request.

jonjensen commented:
@kamilc I saw you added some more commits. Are you all done with this now, or are you still working on it?

kamilc commented:
@jonjensen I've added the header table yesterday - sorry about that. I've just also fixed one error in the description of the cosine. I'm finished now :)

jonjensen commented:
@kamilc Very nice, in-depth work on this! It's now live.
